components/About/about.scss

```diff
@@ -0,0 +1,383 @@
+.header {
```

Comment:
look through this file, you still have minor formatting issues

Reply:
On it

Reply:
Resolved